### PR TITLE
V22F-666 change order menus

### PR DESCRIPTION
**Pull request by sjoeman1**
This PR changes the order of the menus according to the layout we made.
It also makes the cancel and apply buttons consistant on every screen, adds some backgrounds to screens that had missing backgrounds.
It also deletes the raceselectmenu and the racemanmenu and all the menus attached to those

### CODE REVIEW

**Review by larsdekwant — CHANGES_REQUESTED** [review]
According to the diagram the controls menu should be in the options menu, but personally I like the format like this. Either update the diagram (in discussion with the others) or move the menu.

Missing backgrounds:
- save to database menu
- end of experiment menu
- delete data menu

All of the buttons on the menu's above are also slightly too small for the text inside.

**Review by suppergerrie2 — APPROVED** [review]

File: speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/racescreens/racenexteventmenu.cpp [hunk 1]
```diff
@@ -111,15 +111,15 @@ RmNextEventMenu(void)
...
+	GfuiMenuCreateButtonControl(rmScrHandle, menuXMLDescHdle, "AbandonButton", rmScrHandle, GfuiScreenActivate);
 
 	// Close menu XML descriptor.
 	GfParmReleaseHandle(menuXMLDescHdle);
 	
 	// Register keyboard shortcuts.
 	GfuiMenuDefaultKeysAdd(rmScrHandle);
 	GfuiAddKey(rmScrHandle, GFUIK_RETURN, "Start Event", NULL, rmStateManage, NULL);
-	GfuiAddKey(rmScrHandle, GFUIK_ESCAPE, "Abandon", RmRaceSelectMenuHandle, GfuiScreenActivate, NULL);
+	GfuiAddKey(rmScrHandle, GFUIK_ESCAPE, "Abandon", rmScrHandle, GfuiScreenActivate, NULL);
```
Comment (larsdekwant) [inline speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/racescreens/racenexteventmenu.cpp:122]: // SIMULATED DRIVING ASSISTANCE

File: speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/optionsmenu.cpp [hunk 2]
```diff
@@ -57,6 +58,17 @@ onSoundMenuActivate(void * /* dummy */)
     GfuiScreenActivate(SoundMenuInit(MenuHandle));
 }
 
+// SIMULATED DRIVING ASSISTANCE: added control menu
+static void onControlMenuActivate(void* /* dummy */)
```
Comment (larsdekwant) [inline speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/optionsmenu.cpp:62]:
```suggestion
static void OnControlMenuActivate(void* /* dummy */)
```

File: speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/racescreens/racenexteventmenu.cpp [hunk 3]
```diff
@@ -111,15 +111,16 @@ RmNextEventMenu(void)
 
 	// Create Start and Abandon buttons.
 	GfuiMenuCreateButtonControl(rmScrHandle, menuXMLDescHdle, "StartButton", NULL, rmStateManage);
-	GfuiMenuCreateButtonControl(rmScrHandle, menuXMLDescHdle, "AbandonButton", RmRaceSelectMenuHandle, GfuiScreenActivate);
+    // SIMULATED DRIVING ASSISTANCE: replaced rmRaceSelectMenuHandle to rmScrHandle
+	GfuiMenuCreateButtonControl(rmScrHandle, menuXMLDescHdle, "AbandonButton", rmScrHandle, GfuiScreenActivate);
```
Comment (larsdekwant) [inline speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/racescreens/racenexteventmenu.cpp:115]: remove tab

**Review by larsdekwant — CHANGES_REQUESTED** [review]
- Remove the black-box log files from the PR. (the PREFFERED ACTIONS files)
- Fix small code style issues

**Review by larsdekwant — APPROVED** [review]

**Pull request merged**